apps/docs/components/collapsible/Collapsible.tsx

```diff
@@ -3,7 +3,7 @@
 import { CollapseIcon, Icon } from "@/components/icon";
 import clsx from "clsx";
 import type { ReactNode } from "react";
-import { Button, composeRenderProps, UNSTABLE_Disclosure as Disclosure, UNSTABLE_DisclosurePanel as DisclosurePanel, type DisclosureProps } from "react-aria-components";
+import { Button, composeRenderProps, Disclosure, DisclosurePanel, type DisclosureProps } from "react-aria-components";
```

Comment:
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/bd22668d-3211-4ee3-8998-812c53f75c09">
We should have opened sections in the Storybook not only closed Disclosure.